ext/tls/lib.rs

```diff
@@ -156,6 +162,52 @@ pub fn create_client_config(
   Ok(tls_config)
 }
 
+fn load_certs(reader: &mut dyn BufRead) -> Result<Vec<Certificate>, AnyError> {
```

Comment:
These functions are all copied over from `ext/net`. Can you export them in `ext/tls` and remove them from `ext/net` (instead importing from `ext/tls`)?

Reply:
Yep, can do! Should I bump tls version (0.1.1 -> 0.1.2) and update net's dependency on tls?

Reply:
Nope, not necessary (but thanks!). We'll do the releasing + publishing just before the next release, during the release process :-)